script/chart-museum.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# Copyright 2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+CM_PORT=${CM_PORT:-8090}
+CM_USER=${CM_USER:-"admin"}
+CM_PWD=${CM_PWD:-"password"}
+CHART_MUSEUM_NS=${CHART_MUSEUM_NS:-"chart-museum"}
+CM_VERSION=${CM_VERSION:-"3.9.0"}
+
+# Pull a Bitnami chart to a local TGZ file
```

Comment:
```suggestion
# Pull a Bitnami chart to a local TGZ file
# Arguments:
#   $1: Chart name
#   $2: Chart version
```

Reply:
I like to add local variables clearly visible with params, see some lines below:
```bash
local CHART_NAME=$1
local CHART_VERSION=$2
```

But agree, it does not cause harm to explicitly add the comments.